cid/helpers/quicksight/dashboard.py

```diff
@@ -100,53 +101,59 @@ def status(self) -> str:
+        cid_print(f"  <BOLD>Status:<END>    {'<GREEN>' + self.status + '<END>' if self.health else '<RED>' + self.status + '<END>'}")
+
         if self.status_detail:
-            print(f"  Status detail: {self.status_detail}")
-            
-        cid_version = None
-        cid_version_latest = None
-        
+            cid_print(f"  <BOLD>Status detail:<END> {self.status_detail}")
+
+        cid_version = "N/A"
+        cid_version_latest =  "N/A"
+
         try:
             cid_version = self.deployedTemplate.cid_version
         except ValueError:
             logger.debug("The cid version of the deployed dashboard could not be retrieved")
-            cid_version = "N/A"
 
         try:
-            cid_version_latest = self.sourceTemplate.cid_version if isinstance(self.sourceTemplate, CidQsTemplate) else "N/A"
+            if isinstance(self.sourceTemplate, CidQsTemplate):
+                cid_version_latest = self.sourceTemplate.cid_version
         except ValueError:
             logger.debug("The latest version of the dashboard could not be retrieved")
-            cid_version_latest = "N/A"
-            print(f"  CID Version     {cid_version}")
-            print(f"  TemplateVersion {self.deployed_version} ")
 
         if self.latest:
-            logger.debug("The dashboard is up-to-date")
-            logger.debug(f"CID Version {cid_version}")
-            logger.debug(f"TemplateVersion {self.deployed_version} ")
+            cid_print(f"  <BOLD>Version:<END>   <GREEN>{cid_version}<END> (latest)")
+            cid_print(f"  <BOLD>VersionId:<END> <GREEN>{self.deployed_version}<END> (latest)")
         else:
-            print(f"  CID Version     {str(cid_version): <6} --> {str(cid_version_latest): <6}")
-            print(f"  TemplateVersion {str(self.deployed_version): <6} --> {str(self.latest_version): <6}")
-
             logger.debug("An update is available")
-            logger.debug(f"CID Version      {str(cid_version): <9} --> {str(cid_version_latest): <6}")
-            logger.debug(f"TemplateVersion  {str(self.deployed_version): <9} -->  {str(self.latest_version): <6}")
-            
+            cid_print(f"  <BOLD>Version:<END>   <YELLOW>{str(cid_version): <8} --> {str(cid_version_latest): <8}<END>")
+            cid_print(f"  <BOLD>VersionId:<END> <YELLOW>{str(self.deployed_version): <8} --> {str(self.latest_version): <8}<END>")
+
+        cid_print('  <BOLD>Owners:<END>')
+        try:
+            permissions = self.qs.get_dashboard_permissions(self.id)
+            for permission in permissions:
+                if 'quicksight:UpdateDashboardPermissions' in permission["Actions"]:
+                    cid_print('    ' + permission["Principal"].split('user/default/')[-1])
+        except Exception as exc:
+            if "AccessDenied" in str(exc):
+                cid_print('     <RED>AccessDenied<END>')
```

Comment:
If comment at 141 is vali myabe add here which permission exactly is expected, else this error might be a bit more to generic.

Reply:
This access denied can accure when reading. The L141 just checks permissions of user on the dashboard